src/app/header/components/search-bar/search-bar.component.html

```diff
@@ -8,10 +8,16 @@
       class="w-100"
       name="search"
       type="text"
-      #searchInput
+      formControlName="search"
+      (input)="onChange()"
```

Comment:
You don't need to use this one to listen changes on a field from a form the way yo have it.

Check this https://www.tektutorialshub.com/angular/valuechanges-in-angular-forms/

Reply:
Changes made!